### DISCUSSION TITLE
Нет выплат за плотинг

### DISCUSSION THREAD

**Discussion by ShishaMax** [Chia-Network/chia-blockchain#8272] [Support]
Подключился к пулу https://pool.poolchia.com, все синхронизировалась и заработало. С 26 августа перестало поступать вознаграждение за плотинг. Куда можно обратится?

**Comment by loppefaaret** [comment]
ive heard that poolchia.com has changed payout structure! please follow up with poolchia.com on their discord where they announce such changes
https://discord.gg/vB3wqn2dAN

**Comment by BrandtH22** [comment]
@ShishaMax , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
